pkg/k8s/client.go

```diff
@@ -21,6 +22,9 @@ const (
 	getPodListSleepTimeMilliseconds = 300
 )
```

Comment:
First of all - Thank you @yarinm for raising the issue and this PR.

Reply:
@kkmsft I went over your comments and resolved the issues, let me know what you think.